ovos_stt_http_server/routers/openai_whisper.py

```
@@ -0,0 +1,192 @@
+        # STT text so the request still succeeds (degraded behaviour).
+        output_lang = source_lang
+        if translate_to_en and text:
+            translator = _get_translator()
+            if translator is not None:
+                try:
+                    text = translator.translate(text, target="en",
+                                                source=None if source_lang == "auto"
+                                                else source_lang)
+                    output_lang = "en"
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)